components/server/src/workspace/workspace-starter.ts

```diff
@@ -125,7 +125,7 @@ import { WorkspaceClasses, WorkspaceClassesConfig } from "./workspace-classes";
 import { EntitlementService } from "../billing/entitlement-service";
 import { BillingModes } from "../../ee/src/billing/billing-mode";
 import { AttributionId } from "@gitpod/gitpod-protocol/lib/attribution";
-import { BillingServiceClient, BillingServiceDefinition, System } from "@gitpod/usage-api/lib/usage/v1/billing.pb";
```

Comment:
I think you can remove the `billingservice` field as well.

Reply:
Good spot. Done.